src/security/authenticated_user.js

```diff
@@ -50,6 +50,9 @@ var AuthenticatedUser = mCAP.User.extend({
     mCAP.User.prototype.setReferencedCollections.apply(this, [attr]);
 
     if (attr.preferences && !(attr.preferences instanceof mCAP.Model) && this.get('preferences')) {
+      attr.preferences = _.omit(attr.preferences, function(value, key) {
+        return key.match('^custom') && value === '';
```

Comment:
Actually `.match()` expects a regex. Probably match will cast a string to regex internally. Anyway i would prefer `key.match(/^custom/)`